server/src/dynamodb.ts

```diff
@@ -89,40 +68,27 @@ export class Dynamo implements Database {
       ReturnValues: 'UPDATED_NEW',
     };
 
-    try {
-      const resp = await this.dynamoDb.update(params).promise();
-      console.log('Successfully updated item', JSON.stringify(resp));
-      return projectId;
-    } catch (err) {
-      console.log(`Unable to update listoProject with id ${projectId}`, err);
-      throw err;
-    }
+    const resp = await this.db.update(params).promise();
+    console.log('Successfully updated item', JSON.stringify(resp));
+    return projectId;
   }
 
-  public async getProject(projectId: string): Promise<DatabaseModel> {
-    try {
-      const params = {
-        TableName: tableName,
-        Key: {
-          id: projectId,
-        },
-      };
+  public async find(projectId: string): Promise<DatabaseModel> {
+    const params = {
+      TableName: tableName,
+      Key: {
+        id: projectId,
+      },
+    };
 
-      const data = await this.dynamoDb.get(params).promise();
+    const data = await this.db.get(params).promise();
 
-      if (!Object.keys(data).length) {
-        throw 'Project not found';
-      }
+    if (!Object.keys(data).length) {
+      throw 'Project not found';
+    }
 
-      const dbObject = <DatabaseModel>data.Item;
+    const dbObject = <DatabaseModel>data.Item;
 
-      return dbObject;
-    } catch (err) {
```

Comment:
Why did you remove the error handling?

Reply:
The error isn't being handled here though, it's just been logged and re-thrown. Would suggest an express error middleware for logging + returning a "nice" error. The request context (e.g. what method) will give context for what action is being performed, and the stack trace will tell you exactly where to look.